src/lib/index.ts

```diff
@@ -94,9 +93,34 @@ function createWeb3Stores() {
 
 	return {
 		subscribe,
-		connectProvider,
-		disconnectProvider
+		connect,
+		disconnect
 	};
 }
 
-export const web3Stores = createWeb3Stores();
+export function createContractStore(
+	address: string,
```

Comment:
I would here also already use the `Account` type to be more explicit what kind of address
```suggestion
	address: Account,
```